pkgs/development/python-modules/bencoder/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib
+, python3Packages
+, fetchPypi
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "bencoder";
+  version = "0.2.0";
+
```

Comment:
format is missing. If the packaging format is setuptools then we can likely drop it below.